src/dotnet/Fable.Compiler/Transforms/FSharp2Fable.fs

```diff
@@ -232,6 +232,45 @@ let private transformUnionCaseTest (com: IFableCompiler) (ctx: Context) r
         return Fable.Test(unionExpr, kind, r)
   }
 
+let private isRasingMatchFailureExpr (expr: FSharpExpr) = 
+    match expr with 
+    | BasicPatterns.Call(None, methodInfo, [ ], [unitType], [value]) ->   
+        match methodInfo.FullName with 
+        | "Microsoft.FSharp.Core.Operators.raise" -> 
+            match value with 
+            | BasicPatterns.NewRecord(recordType, [ BasicPatterns.Const (value, valueT) ; rangeFrom; rangeTo ]) -> 
+                match recordType.TypeDefinition.FullName with 
+                | "Microsoft.FSharp.Core.MatchFailureException"-> Some (value.ToString())
+                | _ -> None
+            | _ -> None 
+        | _ -> None 
+    | _ -> None  
+
+/// Creates a "throw Error({errorMessage})" expression
+let private createThrowErrorExpr (errorMessage: string) range : Fable.Expr = 
```

Comment:
There's already a similar helper in [Replacements.Helpers.error](https://github.com/fable-compiler/Fable/blob/d1ef91e4a0bf8311f21e42acd3245e178a3c0b2d/src/dotnet/Fable.Compiler/Transforms/Replacements.fs#L97-L98), you could use it to avoid code duplication.